_posts/2024-01-12-kasten-openshift-monitoring.markdown

```diff
@@ -0,0 +1,815 @@
+---
+layout: post
+title: Using Openshift Monitoring for Kasten
+description: Openshift comes with an advanced monitoring stack which is fully compatible with Kasten, in this blog post I will sho how to use it.
+date: 2024-01-12 10:17:00 +0000
+author: michaelcourcy
+image: '/images/posts/2024-01-12-kasten-openshift-monitoring/monitoring.jpg'
+image_caption: 'Using Openshift monitoring for Kasten'
+tags: [Tutorial, Monitoring, Backup, Prometheus, Thanos]
+featured:
+---
+# Using openshift monitoring for Kasten
+
+[Openshift monitoring](https://docs.openshift.com/container-platform/4.12/monitoring/monitoring-overview.html) allow you to monitor user metrics, and allow external services like Zabbix or Dynatrace to access those metrics in a secure way. 
+
+Most of the time, before you start to add Kasten to openshift there is already an ITSM stack in the company which is working with openshift monitoring. All you want to do is include Kasten in Openshift Monitoring to leverage this ITSM stack.
```

Comment:
```suggestion
Typically, before you deploy Kasten to OpenShift, there is already an ITSM stack configured to use OpenShift monitoring. Rather than onboarding Kasten into the ITSM directly, you can include Kasten events within Openshift Monitoring to leverage this already-established integration. This can save you time and effort and be repeated across clusters, to help standardize and simplify monitoring and reporting.
```

Reply:
Recommend rewriting:
Before deploying Kasten on OpenShift, it is typical than an existing Information Technology Service Management (ITSM) stack has been configured to use OpenShift monitoring.
...
By repeating this across other OpenShift clusters, monitoring is standardized, reporting is simplified, and effort is saved.